.github/workflows/ci.yml

```diff
@@ -51,7 +51,9 @@ jobs:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v1
-    - uses: actions/setup-ruby@v1
+    - uses: ruby/setup-ruby@v1
+    - run: gem install asciidoctor
+    - run: Rename-Item -Path C:\msys64 -NewName msys64-tmp -Force
```

Comment:
Minor nit: I wonder if it would be useful to add a short comment here to explain the rename. Reading this line without the context of this PR would make me 🤔 in a few months 🤣

Reply:
Good idea, thanks!  Done!